opensafely/pull.py

```diff
@@ -28,13 +31,20 @@ def add_arguments(parser):
         action="store_true",
         help="Update docker images even if not present locally",
     )
+    parser.add_argument(
+        "--preseed",
```

Comment:
I don't know what the correct terms are here but I'm pretty sure `pre` (it's a prefix but is that the right term when talking about language?) should be hyphenated (eg `--pre-seed`), not gating the review on it though.

Reply:
Hmm, it's commonly `preseed` in debian/ubuntu world, e.g. https://help.ubuntu.com/lts/installation-guide/s390x/apb.html

I wonder if `--project` would be better anyway?

Reply:
Have changed to `--project`.